mlbench_core/dataset/nlp/pytorch/language_model.py

```diff
@@ -0,0 +1,50 @@
+import math
```

Comment:
Isn't it less confusing if we name the dataloader classes like the dataset they represent (i.e. `wikitext2_dataset.py`), as in other benchmarks ? Cause we now have 3 different datasets (including transformer) in `nlp/`